packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -0,0 +1,182 @@
+import { Component, EventEmitter, Inject, Input, OnChanges, OnInit, Output, SimpleChanges } from "@angular/core";
+import { TranslateService } from "@ngx-translate/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+/**
+ * Component to display dropdown list based on the options passed as parameters. The dropdown component is based
+ * on the Angular Material MD-SELECT.
+ */
+@Component({
+	selector: "stark-dropdown",
+	templateUrl: "./dropdown.component.html"
+})
+export class StarkDropdownComponent implements OnInit, OnChanges {
+	@Input()
+	public defaultBlank?: boolean;
+	/**
```

Comment:
Can you add a blank line between the JSDocs and the previous line of code to make it more readable?

The same for the other JSDocs in this class ;)